### DISCUSSION TITLE
Help: User Dropend in sidebar (maybe a bug)

### DISCUSSION THREAD

**Discussion by vjmartins** [tabler/tabler#1600] [Q&A]
Hi,

First, awesome job. I work in a ONG and use tabler to create a internal site. 👍 

Im using Combined Layout, but i cant use a Dropend in sidebar.

Can you help me?

Sorry my english. 😅 

Cards is a Dropend menu.
![prin1](https://user-images.githubusercontent.com/827692/223449441-d137b396-28ad-48fa-bfca-9a8219970d4a.png)

**Comment by vjmartins** [comment]
Hi.
I'm trying to do something like this
https://bootstrap-menu.com/demos/sidebar-nav-submenu.html

**Comment by vjmartins** [comment]
Is opening to me.  I send a screenshot
![1](https://user-images.githubusercontent.com/827692/227510843-cce14fe7-c6c2-4fcf-818e-31d76342a24b.png)

**Comment by cavasinf** [comment]
There's no way to do this natively today.

The "vertical" menu works with "drawers".
https://preview.tabler.io/layout-combo.html

"Horizontal" menu works with "floating" submenu.
![image](https://user-images.githubusercontent.com/25293190/234248856-5e3a819f-fa4f-4a6e-afa4-3ea6f88685e5.png)
